jupyter/trustyai/container/Pipfile

```diff
@@ -19,12 +17,15 @@ jupyterlab-lsp = "~=3.10.2"
 jupyterlab-tabnine = "~=0.0.24"
 jupyterlab-widgets = "~=3.0.5"
 kafka-python = "~=2.0.2"
+matplotlib = "~=3.6.3"
 numpy = "~=1.24.1"
+pandas = "~=1.5.3"
 plotly = "~=5.13.0"
 scikit-learn = "~=1.2.1"
 scipy = "~=1.10.0"
 streamlit = "~=1.17.0"
-# ---
+
+#  Parent image requirements to maintain cohesion
```

Comment:
If you did not modify the Makefile, the base image you're using is the "datascience" one, which already includes all the above packages. So this line should go before them, where you deleted it.